src/tests/end-to-end/common/scan-for-accessibility-issues.ts

```diff
@@ -17,13 +17,10 @@ export async function scanForAccessibilityIssues(
     await injectAxeIfUndefined(page);
     const axeResults = (await page.evaluate(
         options => {
-            return axe.run(
-                { include: [options.selector] } as ElementContext,
-                {
-                    runOnly: { type: 'tag', values: ['wcag2a', 'wcag21a', 'wcag2aa', 'wcag21aa'] },
-                    rules: options.rules,
-                } as ElementContext,
-            );
+            return axe.run({ include: [options.selector] } as ElementContext, {
+                runOnly: { type: 'tag', values: ['wcag2a', 'wcag21a', 'wcag2aa', 'wcag21aa'] },
+                rules: options.rules,
+            });
```

Comment:
This is a no-op change, required because upstream `axe.run` typings became more strict in this update